save-frontend/src/main/kotlin/com/saveourtool/save/frontend/components/views/usersettings/right/Delete.kt

```diff
@@ -0,0 +1,12 @@
+/**
+ * rendering for Delete User management card
+ */
+
+package com.saveourtool.save.frontend.components.views.usersettings.right
+
+import com.saveourtool.save.frontend.components.views.usersettings.SettingsProps
+import react.FC
+
+val delete: FC<SettingsProps> = FC { props ->
```

Comment:
```suggestion
val delete: FC<SettingsProps> = FC { props ->
```

Suggestion to use a better name